convert2rhel/actions/system_checks/eus.py

```diff
@@ -0,0 +1,52 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import datetime
+import logging
+
+from convert2rhel import actions
+from convert2rhel.systeminfo import EUS_MINOR_VERSIONS, system_info
+from convert2rhel.toolopts import tool_opts
+
+
+logger = logging.getLogger(__name__)
+
+
+class EusSystemCheck(actions.Action):
+    id = "EUS_SYSTEM_CHECK"
+
+    def run(self):
+        """Inhibit the conversion when we are not able to handle UEFI."""
+        super(EusSystemCheck, self).run()
+
+        current_version = "%s.%s" % (system_info.version.major, system_info.version.minor)
+        if current_version == "8.8":
```

Comment:
I don't think this will be a blocker, but I'm wondering if we should modify this a bit to actually give the warning on another EUS minor versions instead of just 8.8 (for example, 8.10, 9.2, 9.4, ...)

Maybe we could skip the first item in the dictionary in the `EUS_MINOR_VERSIONS` and check if the `current_version` is in `EUS_MINOR_VERSIONS.keys()`.

For example, that might work: 

```python
>>> current_version = "8.8"
>>> EUS_MINOR_VERSIONS.update({"8.10": "2023-11-14"})
>>> EUS_MINOR_VERSIONS
{'8.6': '2022-11-09', '8.8': '2023-11-14', '8.10': '2023-11-14'}
>>> eus_versions = list(EUS_MINOR_VERSIONS.keys())[1:]
>>> eus_versions
['8.8', '8.10']
>>> current_version in eus_versions
True
```

Reply:
But as I mentioned, this might not be needed for now and we probably can leave this for another release

Reply:
Good idea, would save us the trouble of having to update in the future